[2023-09-02T11:02:06.671+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: website_crawler.crawl_task manual__2023-09-02T11:01:59.414099+00:00 [queued]>
[2023-09-02T11:02:06.703+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: website_crawler.crawl_task manual__2023-09-02T11:01:59.414099+00:00 [queued]>
[2023-09-02T11:02:06.709+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2023-09-02T11:02:06.750+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): crawl_task> on 2023-09-02 11:01:59.414099+00:00
[2023-09-02T11:02:06.777+0000] {standard_task_runner.py:57} INFO - Started process 64 to run task
[2023-09-02T11:02:06.794+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'website_crawler', 'crawl_task', 'manual__2023-09-02T11:01:59.414099+00:00', '--job-id', '258', '--raw', '--subdir', 'DAGS_FOLDER/crawl_data_using_BeautifulSoup.py', '--cfg-path', '/tmp/tmpxyw3w9ca']
[2023-09-02T11:02:06.807+0000] {standard_task_runner.py:85} INFO - Job 258: Subtask crawl_task
[2023-09-02T11:02:07.044+0000] {task_command.py:415} INFO - Running <TaskInstance: website_crawler.crawl_task manual__2023-09-02T11:01:59.414099+00:00 [running]> on host 9b7e267dee3b
[2023-09-02T11:02:07.442+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='website_crawler' AIRFLOW_CTX_TASK_ID='crawl_task' AIRFLOW_CTX_EXECUTION_DATE='2023-09-02T11:01:59.414099+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-02T11:01:59.414099+00:00'
[2023-09-02T11:03:03.593+0000] {local_task_job_runner.py:288} WARNING - DagRun timed out after 0:01:03.317087.
[2023-09-02T11:03:08.730+0000] {local_task_job_runner.py:288} WARNING - DagRun timed out after 0:01:08.454121.
[2023-09-02T11:03:08.737+0000] {local_task_job_runner.py:294} WARNING - State of this instance has been externally set to skipped. Terminating instance.
[2023-09-02T11:03:08.747+0000] {process_utils.py:131} INFO - Sending 15 to group 64. PIDs of all processes in the group: [64]
[2023-09-02T11:03:08.750+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 64
[2023-09-02T11:03:08.871+0000] {taskinstance.py:1632} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-09-02T11:03:09.015+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=64, status='terminated', exitcode=0, started='11:02:06') (64) terminated with exit code 0
